ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -137,6 +141,7 @@ function Body({ hasReadAccess, isFeatureFlagEnabled }: BodyProps): ReactElement
                     <Route path={networkPath} component={AsyncNetworkGraphPage} />
                     <Route path={violationsPath} component={AsyncViolationsPage} />
                     <Route path={compliancePath} component={AsyncCompliancePage} />
+                    <Route path={complianceEnhancedPath} component={AsyncComplianceEnhancedPage} />
```

Comment:
In theory, the route also needs conditional rendering, but skip it because I need to refactor all of this to support declarative configuration of feature flags and permission resources for Body and NavigationSidebar.